src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -18,54 +18,104 @@
 //==============================================================================
 
 #include <BeastConfig.h>
+#include <ripple/app/misc/LoadFeeTrack.h>
 #include <ripple/basics/contract.h>
 #include <ripple/basics/Log.h>
 #include <ripple/basics/mulDiv.h>
-#include <ripple/core/LoadFeeTrack.h>
 #include <ripple/core/Config.h>
+#include <ripple/ledger/ReadView.h>
 #include <ripple/protocol/STAmount.h>
 #include <ripple/protocol/JsonFields.h>
 
 namespace ripple {
 
+bool
+LoadFeeTrack::raiseLocalFee ()
```

Comment:
Nope.  `raiseLocalFee()` and `lowerLocalFee()` just changed their location the file.